sdks/python/apache_beam/io/textio_test.py

```diff
@@ -101,17 +100,19 @@ def write_data(
     return f.name, [line.decode('utf-8') for line in all_data]
 
 
-def write_pattern(lines_per_file, no_data=False):
+def write_pattern(lines_per_file, no_data=False, return_filenames=False):
```

Comment:
It might be nice to change all of the uses of the function, instead of conditionally changing the return type. But I won't block the PR on this : ) LGTM

Reply:
I don't believe any of the other tests care about the filenames.